lib/src/main/java/org/team100/lib/commands/drivetrain/manual/ManualWithFullStateHeading.java

```diff
@@ -132,7 +132,15 @@ public FieldRelativeVelocity apply(SwerveState state, DriverControl.Velocity twi
         if (m_thetaSetpoint == null) {
             // TODO: to avoid overshoot, maybe pick a setpoint that is feasible without
             // overshoot?
-            updateSetpoint(yawMeasurement, yawRate);
+            double accel;
```

Comment:
I was hoping this could fix the todo by creating a setpoint which changes slowly with the max acceleration of the robot, which is what hte code I added should do.

Reply:
i think that comment should actually be about adjusting the *goal*, not the setpoint.  the setpoint needs to be whatever the current robot state is, if you want to avoid some sort of jerky transition.